cmd/rancher_release/set_kdm_branch_references.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"os"
+
+	"github.com/rancher/ecm-distro-tools/release/rancher"
+	"github.com/urfave/cli"
+)
+
+func setKDMBranchReferencesCommand() cli.Command {
+	return cli.Command{
+		Name:  "set-kdm-branch-references",
+		Usage: "set kdm branch references in files",
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:     "rancher-fork-dir",
```

Comment:
Does this take the name of the director or expect the full path to the directory? If it's the full path, can you rename the flag to `fork-path`?